### PR TITLE
[Yosegi-12] Documentation on Hive's table

### PR DESCRIPTION
**Pull request by koijima**
### What is the necessity of this update? What is updated?
I created a document on Hive's table creation.
Quickstart only creates a table, but in this document I describe the settings and the corresponding types.

### CODE REVIEW

File: docs/create_table.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+|:-----------|:------------:|
+|ARRAYS|**true**|
+|MAPS|**true**|
+|STRUCT|**true**|
+|UNION|**true**|
+
+## Cast of type
+If the defined type is different from the data type, cast as much as possible.
+If it can not be done, it is NULL.
+
```
Comment (yohto) [inline docs/create_table.md]: Yosegi execute data type casting when there is a difference between two data types, one is from estimated based on input data, and the table defines the other. If Yosegi cannot data type casting, it treats the data as NULL.

File: docs/create_table.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+## Compression class
+Please see the list of supported compression formats.
+
+## Make complicated data correspond to vectorization
+Hive does not work on complex schema tables in vectorization and pushdown in some cases.
+Data handled by Hive can be processed efficiently by using a flat schema, but it is not always possible to define a flat schema for data design.
```
Comment (yohto) [inline docs/create_table.md]: Hive can handle efficiently the data that is defined by a flat schema. However, it is not always possible to use a flat schema for data design.

File: docs/create_table.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## Cast of type
+If the defined type is different from the data type, cast as much as possible.
+If it can not be done, it is NULL.
+
+## Compression class
+Please see the list of supported compression formats.
+
+## Make complicated data correspond to vectorization
+Hive does not work on complex schema tables in vectorization and pushdown in some cases.
```
Comment (yohto) [inline docs/create_table.md]: Hive cannot work on complex schema tables in vectorization and pushdown in some cases.

File: docs/create_table.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+Data handled by Hive can be processed efficiently by using a flat schema, but it is not always possible to define a flat schema for data design.
+
+In Yosegi, it is possible to process complicated schemas as a flat schema by changing array expansion and node configuration.
```
Comment (yohto) [inline docs/create_table.md]: On the other hand, Yosegi can change complex schemas as a flat schema using array expansion and node configuration.

**Comment by koijima** [comment]
@yohto 
Thank you!
I updated the sentences.

File: docs/create_table.md (hunk 1 above)
Comment (yohto) [reply docs/create_table.md]: updated (sorry for modifying comment not adding)

**Comment by koijima** [comment]
Has been updated.
Thank you.

**Comment by yohto** [comment]
LSTM

**Pull request merged**